src/views/emails/email2022.js

```diff
@@ -208,7 +208,7 @@ const getStyles = () => `
       background-position: 50%;
       background-image: url('${images.logoLight}');
       background-repeat: no-repeat;
-      background-size: 240px 50px;
+      background-size: 343px 50px;
```

Comment:
Probably not a huge issue, but I think previously-sent emails still refer to the same URL that would now start to serve images at a different size, but with the previous CSS.

Reply:
Oh this is a good point... it might be safest to just make the new images the same dimension as the old.

Reply:
OK I resized the images to match the dimensions of the old so it should be more drop-in - thanks!